std-bits/snowflake/src/main/java/org/enso/snowflake/SnowflakeIntegerColumnMaterializer.java

```diff
@@ -56,12 +56,15 @@ private boolean fitsInLong(BigInteger bigInteger) {
   }
 
   @Override
-  public void appendNoGrow(Object o) {
+  public void append(Object o) {
+    if (currentSize >= capacity()) {
+      grow();
```

Comment:
```suggestion
      ensureSpaceToAppend();
```

I think this is what this method is called elsewhere

Reply:
I'll redo this builder in the next step - it should be more like the other "inferred" builders.